### PR TITLE
Fix osx master build

### PR DESCRIPTION
**Pull request by mantid-roman**
Removed mentions of the VatesSimpleGui stand alone executable from build/install scripts.

### CODE REVIEW

**Comment by martyngigg** [comment]
@mantid-roman It looks like there are more changes required in the `make_package.rb` script relating to the library names.

**Review by martyngigg — APPROVED** [review]
@mantidproject/gatekeepers This fixes the OSX packaging that had apparently been broken in a few places. The PR on OSX has been rebuilt with `BUILD_PACKAGE=ON` and it now passes.

File: buildconfig/CMake/CppCheckSetup.cmake [hunk 1]
```diff
@@ -30,7 +30,6 @@ if ( CPPCHECK_EXECUTABLE )
         Framework/Kernel/inc
         Vates/VatesAPI/inc
         Vates/VatesSimpleGui/ViewWidgets/inc
-        Vates/VatesSimpleGui/StandAloneExec/inc
```
Comment (quantumsteve) [inline buildconfig/CMake/CppCheckSetup.cmake]: Should these other cppcheck include directories be updated?

Comment (martyngigg) [inline buildconfig/CMake/CppCheckSetup.cmake]: We noticed that too and @mantid-roman fixed them in #20291, which is also ready to go.

**Pull request merged**